translations/en.json

```diff
@@ -493,6 +493,113 @@
+		"no-active-loans": "You have no active borrows.",
+		"tabs": {
+			"new": {
+				"title": "Borrow Synths",
+				"button": {
+					"low-collateral-label": "MINIMUM COLLATERAL IS <0>{{minCollateralAmountString}}</0> <0>{{collateralAsset}}</0>",
+					"low-cratio-label": "C-RATIO TOO LOW",
+					"approve-label": "APPROVE <0>{{collateralAsset}}</0>",
+					"borrow-label": "BORROW <0>{{debtAsset}}</0>",
+					"approving-label": "APPROVING...",
+					"borrowing-label": "BORROWING..."
+				},
+				"debt": {
+					"label": "Borrow"
+				},
+				"collateral": {
+					"label": "Using"
+				},
+				"low-cratio-tip": "You can only borrow at 150% or higher. Your position will be eligible for liquidation if it falls below 120%.",
+				"healthy-cratio-tip": "Ensure your position stays above 120% to prevent liquidation.",
+				"confirm-transaction": {
+					"left-panel-label": "BORROWING",
+					"right-panel-label": "POSTING"
+				}
+			},
+			"list": {
+				"title": "Active Borrows",
+				"types": {
+					"debt": "LOAN",
+					"collateral": "COLLATERAL",
+					"cratio": "C-RATIO",
+					"modify": "MODIFY"
+				}
+			}
+		},
+		"info": {
+			"title": "Borrow Synths using other assets",
+			"subtitle": "Lorem ipsum dolor sit amet, consectetur id feugiat vel quis urna consequat in. Netus arcu massa purus dolor sit.",
+			"learn-more": "Learn more."
+		},
+		"pending-withdrawals": {
+			"title": "Pending Withdrawals",
+			"title-tip": "You will need to claim your ether collateral separately after your loan is closed, withdrawn or liquidated.",
+			"empty": "You have no pending withdrawals.",
+			"claiming": "CLAIMING...",
+			"claim": "CLAIM"
+		},
+		"stats": {
+			"title": "STATS",
+			"asset": "ASSET",
+			"open-interest": "OPEN INTEREST",
+			"total": "TOTAL"
+		},
+		"modify-loan": {
+			"loan-interation-delay": "Please wait <0>{{waitETA}}</0> before reinteracting with the loan",
+			"close": {
+				"left-col-label": "REPAY",
+				"right-col-label": "RECEIVE",
+				"button-labels": {
+					"default": "CLOSE LOAN",
+					"closing": "CLOSING LOAN..."
+				}
+			},
+			"deposit": {
+				"left-col-label": "DEPOSIT",
+				"right-col-label": "NEW TOTAL",
+				"button-labels": {
+					"default": "DEPOSIT",
+					"approve": "APPROVE",
+					"approving": "APPROVING...",
+					"depositing": "DEPOSITING..."
+				}
+			},
+			"withdraw": {
+				"left-col-label": "WITHDRAW",
+				"right-col-label": "REMAINING",
+				"button-labels": {
+					"default": "WITHDRAW",
+					"withdrawing": "WITHDRAWING..."
+				}
+			},
+			"repay": {
+				"left-col-label": "REPAY",
+				"right-col-label": "REMAINING",
+				"button-labels": {
+					"default": "REPAY DEBT",
+					"repaying-fully-error": "CONSIDER CLOSING THE LOAN INSTEAD OF FULL REPAYMENT",
+					"repaying": "REPAYING DEBT..."
+				}
+			},
+			"draw": {
+				"left-col-label": "INCREASE",
+				"right-col-label": "NEW TOTAL",
+				"button-labels": {
+					"default": "INCREASE DEBT",
+					"drawing": "INCREASING DEBT..."
```

Comment:
Hey @vbstreetz we usually don't uppercase translations as this might confuse translators. You can use `uppercase` via css for such situations.

Reply:
Awesome. On it..